turbopack/crates/turbo-tasks-backend/src/database/turbo.rs

```diff
@@ -110,7 +110,7 @@ impl KeyValueDatabase for TurboKeyValueDatabase {
 }
 
 pub struct TurboWriteBatch<'a> {
-    batch: turbo_persistence::WriteBatch<WriteBuffer<'static>, 5>,
+    batch: turbo_persistence::WriteBatch<WriteBuffer<'static>, 10>,
```

Comment:
I did this because otherwise I got a panic due to `family_id` being larger than the `FAMILIES` constant.